lasagne/regularization.py

```diff
@@ -1,3 +1,53 @@
+"""
+Functions to apply regularization to the weights in a network.
+
+We provide functions to calculate the L1 and L2 penalty. Penalty functions
+take a tensor as input and calculate the penalty contribution from that tensor:
+
+.. autosummary::
+    :nosignatures:
```

Comment:
In my opinion it still look a little odd with signatures and `x` doesn't really convey any information. 
I can rename `x` to `tensor` and add the signature?